### PR TITLE
chore: font organization

### PR DESCRIPTION
**Pull request by stephiescastle**
### Checklist

- [x] Include a description of your pull request and instructions for the reviewer to verify your work.
- [x] Link to the issue if this PR is issue-specific.
- [ ] Create/update the corresponding story if this includes a UI component.
- [ ] Create/update documentation. If not included, tell us why.
- [ ] List the environments / browsers in which you tested your changes.
- [ ] Tests, linting, or other required checks are passing.
- [x] PR has an informative and human-readable title
  - PR titles are used to generate the change log in [releases](../releases); good ones make that easier to scan.
  - PRs will be broadly categorized in the change log, but for even easier scanning, consider prefixing with a component name or other useful categorization, e.g., "BaseButton: fix layout bug", or "Storybook: Update dependencies".
- [x] PR has been tagged with a [SemVer](https://semver.org/) label and a [general category label](https://github.com/nasa-jpl/explorer-1/blob/52994b671411f55961d7beb8851d4580ac3f434f/.github/release-drafter.config.yml#L21-L39), or skip-changelog.
  - These tags are used to do the aforementioned broad categorization in the change log and determine what the next release's version number should be.
  - [Release Drafter](https://github.com/marketplace/actions/release-drafter) will attempt to do the category labeling for you! Please double-check its work.

## Description

- Reorganizes fonts to allow consuming projects to load fonts separately. 
- styles.scss no longer imports font-face styles
- if a project wants to include font-face styles, they should use styles-with-fonts.scss
- Unrelated: adjusts `MixinCarousel` to not allow clicking on slides that are "out of view" (partially visible or transparent) as it was failing accessibility tests in some instances.

Note: I did consider making the default styles.scss include fonts and creating a separate file that excludes them, but the way I've done it in this PR is the most efficient code-wise. 

## Breaking Changes
- Sites that are importing `styles.scss` will need to now import fonts separately or use `styles-with-fonts.scss` instead

### Operating System

- [ ] macOS
- [ ] iOS
- [ ] iPadOS
- [ ] Windows

### Browser

- [ ] Chrome
- [ ] Firefox ESR
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

**Review by jamesray — APPROVED** [review]

File: packages/common/src/scss/_typography.scss [hunk 1]
```diff
@@ -1,6 +1,4 @@
 @use 'sass:math';
-@tailwind base;
-@tailwind utilities;
```
Comment (Scotchester) [inline packages/common/src/scss/_typography.scss:3]: Are these just not needed? I don't see them re-added anywhere else.

File: packages/common/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@explorer-1/common",
-  "version": "1.0.5",
```
Comment (Scotchester) [inline packages/common/package.json:3]: Should this maybe be considered a feature, warranting a 1.1.0 version? I don't feel strongly about it, though.

**Review by Scotchester — COMMENTED** [review]
A question borne of my ignorance about the new multi-package structure: As the HTML and Vue packages are now including styles-with-fonts, how does a consumer opt to use styles(-without-fonts)? I gather that the docs for this revamp are still a WIP, but we'll definitely want to remember to explain this there later.

File: packages/common/src/scss/_typography.scss (hunk 1 above)
Comment (stephiescastle) [reply packages/common/src/scss/_typography.scss:3]: They were redundant!

File: packages/common/package.json (hunk 2 above)
Comment (stephiescastle) [reply packages/common/package.json:3]: ah right, I'll adjust that. I think I labeled the PR as a major version. I'll reflect that in the version number.

**Comment by stephiescastle** [comment]
> how does a consumer opt to use styles(-without-fonts)?

It depends on how the user wants to optimize their fonts. For us, we are going with @nuxt/fonts, which we will likely add docs on how to use.

Nuxt-less, users can optimize fonts the way we did originally in www-frontend (load the _fonts.scss file as css on the frontend). We can include a way to generate a 'font-face.css' from the scss file in the docs, but I don't think it's urgent now. I created a [stub issue](https://github.com/nasa-jpl/explorer-1/issues/524) for this, though really we need a stub issue for docs overhaul.

File: packages/common/package.json (hunk 2 above)
Comment (stephiescastle) [reply packages/common/package.json:3]: I changed the PR label and version numbers to reflect a minor version.

**Pull request merged**